File: Docs/CS/Data/Commands/ChannelCommands.cs
namespace DevExpress.DevAV.Chat.Commands {
    public abstract class ChannelCommand {
        public IChannel Channel {
            get;
        }
    }
    //
    public class LogOff : ChannelCommand {
        public LogOff(IChannel channel) 
            : base(channel) {
        }
    }
}

File: Docs/VB/Data/Commands/ChannelCommands.vb
Namespace DevExpress.DevAV.Chat.Commands
    Public MustInherit Class ChannelCommand
        Public ReadOnly Property Channel() As IChannel
    End Class
    '
    Public Class LogOff
        Inherits ChannelCommand

        Public Sub New(ByVal channel As IChannel)
            MyBase.New(channel)
        End Sub
    End Class
End Namespace